src/Components/Aspire.Azure.Messaging.WebPubSub/README.md

```diff
@@ -0,0 +1,139 @@
+# Aspire.Azure.Messaging.WebPubSub
```

Comment:
Can you also update 
* https://github.com/dotnet/aspire/blob/main/src/Components/Aspire_Components_Progress.md
* https://github.com/dotnet/aspire/blob/main/src/Components/Telemetry.md